com.shatteredpixel.shatteredpixeldungeon.yaml

```diff
@@ -0,0 +1,52 @@
+app-id: com.shatteredpixel.shatteredpixeldungeon
+runtime: org.freedesktop.Sdk
+runtime-version: 1.6
+sdk: org.freedesktop.Sdk
+command: com.shatteredpixel.shatteredpixeldungeon.sh
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.openjdk10
+finish-args:
+- --socket=wayland
+- --socket=x11
+- --share=ipc
+- --device=dri
+- --env=PATH=/usr/bin:/app/bin:/app/jre/bin
+- --socket=pulseaudio
+- --persist=.shatteredpixel
+modules:
+- name: openjdk
+  buildsystem: simple
+  build-commands:
+    - /usr/lib/sdk/openjdk10/install.sh
+
+- name: xrandr
+  sources:
+  - type: archive
+    url: https://xorg.freedesktop.org/archive/individual/app/xrandr-1.5.0.tar.gz
+    sha256: ddfe8e7866149c24ccce8e6aaa0623218ae19130c2859cadcaa4228d8bb4a46d
+
+- name: shatteredpixeldungeon
+  buildsystem: simple
+  build-commands:
+  - echo "java -jar /app/share/com.shatteredpixel.shatteredpixeldungeon/com.shatteredpixel.shatteredpixeldungeon.jar" > com.shatteredpixel.shatteredpixeldungeon.sh
+  - install -Dm755 com.shatteredpixel.shatteredpixeldungeon.sh /app/bin/com.shatteredpixel.shatteredpixeldungeon.sh
+  - install -Dm644 com.shatteredpixel.shatteredpixeldungeon.desktop /app/share/applications/com.shatteredpixel.shatteredpixeldungeon.desktop
+  - install -Dm644 com.shatteredpixel.shatteredpixeldungeon.appdata.xml /app/share/metainfo/com.shatteredpixel.shatteredpixeldungeon.appdata.xml
+  - install -Dm644 com.shatteredpixel.shatteredpixeldungeon.png /app/share/icons/hicolor/48x48/apps/com.shatteredpixel.shatteredpixeldungeon.png
+  - install -Dm644 com.shatteredpixel.shatteredpixeldungeon.jar /app/share/com.shatteredpixel.shatteredpixeldungeon/com.shatteredpixel.shatteredpixeldungeon.jar
+  sources:
+  - type: file
+    url: https://github.com/00-Evan/shattered-pixel-dungeon-gdx/releases/download/v0.6.5c/ShatteredPD.Desktop.v0.6.5c.jar
+    sha256: 5328a610b13a10ddf981da3fedbde51576b5fd10cf263f73dd79aac126273c85
+    dest-filename: com.shatteredpixel.shatteredpixeldungeon.jar
+  - type: file
+    path: com.shatteredpixel.shatteredpixeldungeon.desktop
+    sha256: 2257a14706a7a8c29e6676f6b7f37a41d08c33856dbf89ee4e37432eedbc0d3b
+  - type: file
+    path: com.shatteredpixel.shatteredpixeldungeon.appdata.xml
+    sha256: c395710959c413af5fdf5c7a28416e2f062ccdf971bd9a59b0061a3672660eff
```

Comment:
Wrong checksum. We don't need them for local files anyway.

Reply:
Done